### ISSUE TITLE
Recurring Downtime for Host and all Services

### ISSUE THREAD

**Issue by elitretti** [sni/Thruk#1377]
**Describe the bug**
When i try to generate a recurrent Downtime for a Host and all Services it works with some workarounds(for ex. setting a own recurrent Downtime for all Services and the Host) but when a new Service is added to that host, then this Service is not part of that Recurrent Downtime.

So as of now there is no simple or pretty solution, to set a recurrent Downtime for a Host and all Services

**Thruk Version**
Current

**To Reproduce**
1. Go to Hosts and select Wanted host
2. Click on hostname for Quick Commands Dialog
3. Select "Add Downtime"
4. Comment / Start / Enddate...
5. Select Services: Schedule downtime for all Services
6. Check Recurring -> Yes
7.  Then you are on the Regular Recurring Downtime dialog. With no possibility to set a Downtime for a Host and all Services with a longterm Solution.

**Expected behavior**
A Button/API Endpoint to Set a Recurrent Downtime for Host and all Services. <3

**Screenshots**
/

**Desktop (please complete the following information):**
 - Chrome
 - 126.0.6478.116 (Official Build) (64-bit)

**Additional context**
So as of now there is no simple or pretty solution, to set a recurrent Downtime for a Host and all Services

**Comment by ccztux** [comment]
I agree with that. It would be nice to set recurring downtimes for a host and all its services as the gui suggests:

![image](https://github.com/sni/Thruk/assets/11867479/85b1938a-0cf6-45c6-adfa-3cc02fff27b1)


But after clicking the "submit command for x host" button, it is not possible to set a recurring downtime for the host and all of its services:

![image](https://github.com/sni/Thruk/assets/11867479/f46b876d-71ec-4bb3-8c69-3ab7b39dcedc)

**Comment by sni** [comment]
it is usually not required to set a downtime for a host and the services since the hostdowntime includes all services already. What is the use case for this?

**Comment by ccztux** [comment]
Honestly speaking due to a past issue in an old nagios version, where a host downtime has not prevented service problems to be notified, our users are used to set the downtime for the host and all its services. Another more cosmetical issue with the host downtimes for me is, that i can not see this info in the `extinfo.cgi?type=2&host=zet&service=ICMP with ARGS&backend=328c5` view.
